File: 0260-single-number-iii/0260-single-number-iii.cs
public class Solution {
    public int[] SingleNumber(int[] nums) {
      List<int> result=new List<int>();
      for(int i=0;i<nums.Length;i++){
        if(Array.IndexOf(nums,nums[i])==Array.LastIndexOf(nums,nums[i])) result.Add(nums[i]); 
    }
    return result.ToArray(); 
    }
}

File: 0260-single-number-iii/0260-single-number-iii.ts
function singleNumber(nums: number[]): number[] {
    let result=[];
    for(let i=0;i<nums.length;i++){
        if(nums.indexOf(nums[i])==nums.lastIndexOf(nums[i])) result.push(nums[i]); 
    }
    return result;
};